django_celery_fulldbresult/serialization.py

```diff
@@ -0,0 +1,17 @@
+from kombu.serialization import registry as k_registry
+from kombu.exceptions import (EncodeError, DecodeError)
+
+
+def dumps(data):
+    try:
+        content_type, encoding, data = k_registry.dumps(data)
+    except EncodeError as e:
+        raise TypeError(e)
+    return data
+
+
+def loads(data, content_type='application/json', encoding='utf-8'):
```

Comment:
small thing, could you use double quotes instead of single quotes (internal code convention).

and can you add a small docstring to these two public functions?